deployments/pulse3d/services/pulse3d/src/core/config.py

```diff
@@ -3,7 +3,7 @@
 
 config = Config(".env")
 # version used to tag the docker image
-VERSION = "0.4.0"
+VERSION = "0.5.1"
```

Comment:
You will also want to update this version in the manifest and the Makefile where the version is the tag for the ECR images